app/models/experience_point_award.rb

```diff
@@ -15,6 +15,7 @@ class ExperiencePointAward < ApplicationRecord
     :streamingatron, # streaming live
     :glorppy, # daily glorp lottery
     :gloppy, # daily glop lottery
+    :shrimpo_prize, # winning shrimpo
```

Comment:
change this name?

Reply:
i guess it seems a little repetitive to say shrimpo_prize when its already a shrimp. There's only one winner per shrimpo right? maybe it can be :first_prize, :trophy_winner, or whatever??

Reply:
Well the thing is this is used directly in the notification text:
```ruby
  when "experience_point_award"
      "You got #{self.source.amount} #{self.source.award_type} points!"`
```

So I was wondering if "You got 500 shrimpo prize points!" would sound ok or if we should change it

Reply:
in that case maybe just 'shrimpo' is fine